src/server/app.py

```diff
@@ -0,0 +1,78 @@
+from flask import Flask
+import logging
+#from flask_restful import Api, Resource, reqparse
+from flask_cors import CORS
+#from FlaskApp.api_layer.HelloApiHandler import HelloApiHandler
+from flask import abort, render_template, request, redirect, url_for, jsonify, json
```

Comment:
Add these functions to fix the server-side cross-site scripting vulnerability
```suggestion
from flask import abort, render_template, request, redirect, url_for, jsonify, json, make_response, escape
```

Reply:
Ok thanks Axel.